test/extended/router/router.go

```diff
@@ -30,16 +30,16 @@ var _ = g.Describe("[Conformance][Area:Networking][Feature:Router]", func() {
 	// hook
 	g.AfterEach(func() {
 		if g.CurrentGinkgoTestDescription().Failed {
-			exutil.DumpPodLogsStartingWithInNamespace("router", "default", oc.AsAdmin())
+			exutil.DumpPodLogsStartingWithInNamespace("router", "openshift-ingress", oc.AsAdmin())
 			selector, err := labels.Parse("router=router")
 			if err != nil {
 				panic(err)
 			}
-			exutil.DumpPodsCommand(oc.AdminKubeClient(), "default", selector, "cat /var/lib/haproxy/router/routes.json /var/lib//var/lib/haproxy/conf/haproxy.config")
+			exutil.DumpPodsCommand(oc.AdminKubeClient(), "openshift-ingress", selector, "cat /var/lib/haproxy/router/routes.json /var/lib//var/lib/haproxy/conf/haproxy.config")
 		}
 	})
 
-	oc = exutil.NewCLI("router-stress", exutil.KubeConfigPath())
+	oc = exutil.NewCLI("openshift-ingress", exutil.KubeConfigPath())
```

Comment:
I think all the existing tests need to remain compatible with old router deployments for now? That is, do like in other parts of the framework where the correct namespace is detected.

Reply:
That was also my concert, thanks for pointing in the right direction.